kontiki.cabal

```diff
@@ -21,32 +21,33 @@ Source-Repository head
 
 Library
   Exposed-Modules:     Network.Kontiki.Raft
+                     , Data.Kontiki.MemLog
   Other-Modules:       Network.Kontiki.Types
                      , Network.Kontiki.Log
                      , Network.Kontiki.Monad
                      , Network.Kontiki.Raft.Follower
                      , Network.Kontiki.Raft.Candidate
                      , Network.Kontiki.Raft.Leader
                      , Network.Kontiki.Raft.Utils
-  Build-Depends:       base >= 4.8 && < 4.9
-                     , mtl >= 2.2 && < 2.3
-                     , bytestring >= 0.10 && < 0.11
-                     , containers >= 0.5 && < 0.6
-                     , binary >= 0.7 && < 0.8
-                     , lens >= 4.12 && < 4.13
-                     , QuickCheck >= 2.8 && < 2.9
+  Build-Depends:       base
```

Comment:
I really like to have version bounds in dependency definitions. The bounds may be too restrictive (the code is getting quite old...), but just dropping them makes publishing this package to Hackage or usage by others more difficult.